main/src/library/Fixpoint/Stratifier.flix

```diff
@@ -87,10 +87,10 @@ namespace Fixpoint {
     }
 
     def mkDepEdge(dst: HeadPredicate[v], src: BodyPredicate[v]): PrecedenceGraph = match (dst, src) {
```

Comment:
I know there was no comments here already, but add some comments to explain what is happening :)